### PR TITLE
clean up multiome wrapper

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

There are multiple inputs and optional tasks that can be consolidated in the Multiome Inputs:

- GEX and ATAC GTFs - we should be bringing in the GTF used for GEX into the ATAC component. That means we only need one GTF input that is passed as input to both subworkflows.
- Remove the CreateFragmentFile barcodes_in_readnames optional boolean.
- Remove the optional python code that uses the barcodes_in_readnames boolean and make the default the option that assumes barcodes in the CB tag (i.e. barcodes_in_readnmaes) is false.
- Rename the ATAC WDL’s FastqProcess task to FastqProcessATAC.
- Move the ATAC WDL’s FastqProcessATAC to the FastqProcess WDL.
- Import the FastqProcessATAC task from the FastqProcess WDL.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
 "Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
```
Comment (nikellepetrillo) [inline pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:2]: @ekiernan does this look like the right gtf to be using here?

Comment (ekiernan) [reply pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:2]: @nikellepetrillo we just confirmed yesterday that the modified one would be the appropriate one for optimus, not the intronic.

File: pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
-"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
-  "Multiome.barcodes_in_read_name":"false",
+"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/modified_v43.annotation.gtf",
```
Comment (nikellepetrillo) [inline pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:2]: when using the intronic GTF, it fails in optimus (calculateCellMetrics) with:
Expected 2 fields, found 0 fields
Expected 2 fields, found 0 fields

Comment (nikellepetrillo) [reply pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:2]: note to self: change back to original --> gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf

**Review by kevinpalis — APPROVED** [review]
Reviewed in WARP maintenance. Looks good!

**Review by aawdeh — APPROVED** [review]
Looks good!

File: pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json (hunk 1 above)
Comment (nikellepetrillo) [reply pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:2]: here is a successful run using `gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf` https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/eb702f53-2de8-410a-9036-72ca3a190f3e/metadata

**Pull request merged**